pkg/skaffold/schema/latest/config.go

```diff
@@ -524,8 +524,6 @@ type Artifact struct {
 
 	// ArtifactType describes how to build an artifact.
 	ArtifactType `yaml:",inline"`
-
-	WorkspaceHash string `yaml:"-,omitempty"`
```

Comment:
just making sure, this won't break any existing configs right?

Reply:
No, this is not serialized.